generators/client/templates/angular/src/main/webapp/app/blocks/config/uib-pagination.config.ts.ejs

```diff
@@ -22,7 +22,7 @@ import { ITEMS_PER_PAGE } from 'app/shared/constants/pagination.constants';
 
 @Injectable({providedIn: 'root'})
 export class PaginationConfig {
-    constructor(private config: NgbPaginationConfig) {
+    constructor(config: NgbPaginationConfig) {
```

Comment:
why did you made it public? (and why the other are too...)

Reply:
it will localize scope to the method